apis/v1alpha2/tcproute_types.go

```diff
@@ -59,6 +59,11 @@ type TCPRouteStatus struct {
 
 // TCPRouteRule is the configuration for a given rule.
 type TCPRouteRule struct {
+	// Matches are rules for routing traffic to backends based on addresses.
+	//
+	// +optional
+	Matches *TrafficMatches `json:"matches,omitempty"`
```

Comment:
HTTPRoute has a slice here (e.g. `Matches []TrafficMatch`) which expresses an OR condition. I don't see this discussed in the GEP, but is there a good reason to diverge from the existing API pattern?

Reply:
I'm OK with switching to the OR style list of traffic matches, cc @robscott any thoughts?

Reply:
+1. It would be for the best to not deviate from the existing API patterns.

Reply:
Sounds good, this was changed to a list in 19f1646b93f7db2e095771848c30fa1c7088c8ea

Reply:
19f1646b93f7db2e095771848c30fa1c7088c8ea includes documentation updates for this, and 9876eede43b01fa0472336eafc60a6422e0d3219 includes the changes to make this a list :+1: